OloEditor/src/Panels/SceneHierarchyPanel.cpp

```
@@ -12,297 +12,296 @@
+    SceneHierarchyPanel::SceneHierarchyPanel(const Ref<Scene>& context)
+    {
+        SetContext(context);
+    }
+
+    void SceneHierarchyPanel::SetContext(const Ref<Scene>& context)
+    {
+        m_Context = context;
+        m_SelectionContext = {};
+    }
+
+    void SceneHierarchyPanel::OnImGuiRender()
```

Clang (14.0.6)

[warning] method 'OnImGuiRender' can be made static

(readability-convert-member-functions-to-static)